packages/scene-composer/stories/Developer/SceneComposer.stories.mdx

```diff
@@ -24,31 +24,28 @@ export const defaultArgs = {
     COMPOSER_FEATURES.Matterport,
     COMPOSER_FEATURES.TagStyle,
     COMPOSER_FEATURES.AutoQuery,
-  ]
-}
+    COMPOSER_FEATURES.DynamicScene,
```

Comment:
do not add this by default yet. it's not ready

Reply:
done